### PR TITLE
docs: add some notes in getting-started

### PR DESCRIPTION
**Pull request by Gala-R**

### CODE REVIEW

File: docs/en_US/getting-started/getting-started.md [hunk 1]
```diff
@@ -1,113 +1,247 @@
...
-## Running the Operator
+### Prepare the environment
 
-### Prepare
+Before deploying EMQX Operator, please confirm that the following components have been installed:
 
-We use a [cert-manager](https://github.com/jetstack/cert-manager) for provisioning the certificates for the webhook server. You can follow [the cert-manager documentation](https://cert-manager.io/docs/installation/) to install it.
+| Software                | Version Requirements |
+|:-----------------------:|:--------------------:|
+| Kubernetes              | >= 1.20.11           |
```
Comment (Rory-Z) [inline docs/en_US/getting-started/getting-started.md:13]: Maybe need >= 1.24 ?

File: docs/en_US/getting-started/getting-started.md [hunk 2]
```diff
@@ -1,113 +1,247 @@
...
+```bash
+helm repo add emqx https://repos.emqx.io/charts
+helm repo update
+```
 
 ### Install EMQX Operator
 
-1. Install by helm
+```bash
+helm install emqx-operator emqx/emqx-operator --set installCRDs=true --namespace emqx-operator-system --create-namespace
```
Comment (Rory-Z) [inline docs/en_US/getting-started/getting-started.md:39]:
```suggestion
helm install emqx-operator emqx/emqx-operator --namespace emqx-operator-system --create-namespace
```

Comment (Rory-Z) [reply docs/en_US/getting-started/getting-started.md:39]: The default value of `installCRDs` is `true`

File: docs/en_US/getting-started/getting-started.md [hunk 3]
```diff
@@ -1,113 +1,247 @@
...
 
-    Full example please check [`emqx-full.yaml`](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v2alpha1/emqx-full.yaml).
+For detailed explanation of each field, please refer to [v2alpha1-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v2alpha1-reference.md)
 
-    Detailed explanation of each field please check [v2alpha1-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v2alpha1-reference.md)
+2. Check whether the EMQX cluster is ready
 
-2. Check EMQX Custom Resource status
+```bash
+kubectl get emqx emqx -o json | jq ".status.emqxNodes"
```
Comment (Rory-Z) [inline docs/en_US/getting-started/getting-started.md:97]: Check `.status.conditions`

File: docs/en_US/getting-started/getting-started.md [hunk 4]
```diff
@@ -1,113 +1,247 @@
...
 
-    Full example please check [`emqxbroker-full.yaml`](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v1beta3/emqxbroker-full.yaml).
+For detailed explanation of each field, please refer to [v1beta3-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md)
 
-    Detailed explanation of each field please check [v1beta3-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md)
+2. Check whether the EMQX cluster is ready
 
-2. Check EMQX Custom Resource status
+```bash
+kubectl get emqxbroker emqx -o json | jq ".status.emqxNodes"
```
Comment (Rory-Z) [inline docs/en_US/getting-started/getting-started.md:162]: Check `.status.conditions`

File: docs/en_US/getting-started/getting-started.md [hunk 5]
```diff
@@ -1,113 +1,247 @@
...
 
-    Full example please check [`emqxenterprise-full.yaml`](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v1beta3/emqxenterprise-full.yaml).
+For detailed explanation of each field, please refer to [v1beta3-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md)
 
-    Detailed explanation of each field please check [v1beta3-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md)
+2. Check whether the EMQX cluster is ready
 
-2. Check EMQX Custom Resource status
+```bash
+kubectl get emqxenterprise emqx-ee -o json | jq ".status.emqxNodes"
```
Comment (Rory-Z) [inline docs/en_US/getting-started/getting-started.md:217]: Check `.status.conditions`

File: docs/en_US/getting-started/getting-started.md [hunk 6]
```diff
@@ -1,113 +1,247 @@
...
\ No newline at end of file
+
+1. The `MixedProtocolLBService` feature is enabled by default in Kubernetes 1.24 and above. For its documentation, please refer to: [MixedProtocolLBService ](https://kubernetes.io/zh-cn/docs/reference/command-line-tools-reference/feature-gates /#feature-gates-for-alpha-or-beta-features). The `MixedProtocolLBService` attribute allows different protocols to be used within the same Service instance of type `LoadBalancer`. Therefore, if the user deploys the EMQX cluster on Kubernetes and uses the `LoadBalancer` type of Service, there are both TCP and UDP protocols in the Service, please pay attention to upgrading the Kubernetes version to 1.24 or above, otherwise the Service creation will fail.
```
Comment (Rory-Z) [inline docs/en_US/getting-started/getting-started.md:247]: Move this block to `Prepare the environment`

File: docs/directory.json [hunk 7]
```diff
@@ -127,10 +123,6 @@
         {
           "title": "部署 EMQX 集群",
           "path": "getting-started/getting-started"
-        },
-        {
-          "title": "部署 EMQX 插件",
-          "path": "getting-started/install-emqx-prometheus-plugin"
```
Comment (Rory-Z) [inline docs/directory.json:133]: Do we need delete Markdown file ?

File: docs/en_US/getting-started/getting-started.md (hunk 1 above)
Comment (Gala-R) [reply docs/en_US/getting-started/getting-started.md:13]: > Maybe need >= 1.24 ?

OK，i wiil change it

File: docs/en_US/getting-started/getting-started.md (hunk 2 above)
Comment (Gala-R) [reply docs/en_US/getting-started/getting-started.md:39]: > The default value of `installCRDs` is `true`

ok

File: docs/directory.json (hunk 7 above)
Comment (Gala-R) [reply docs/directory.json:133]: > Do we need delete Markdown file ?

Xuebin and I confirmed that this document can be deleted. We have a document that specifically introduces the use of prometheus to collect EMQX indicators, so this document is meaningless.

File: docs/en_US/getting-started/getting-started.md (hunk 4 above)
Comment (Gala-R) [reply docs/en_US/getting-started/getting-started.md:162]: > Check `.status.conditions`

i will change it

File: docs/en_US/getting-started/getting-started.md (hunk 5 above)
Comment (Gala-R) [reply docs/en_US/getting-started/getting-started.md:217]: > Check `.status.conditions`

i will change

File: docs/en_US/getting-started/getting-started.md (hunk 3 above)
Comment (Gala-R) [reply docs/en_US/getting-started/getting-started.md:97]: > Check `.status.conditions`

i will change it

File: docs/en_US/getting-started/getting-started.md (hunk 6 above)
Comment (Gala-R) [reply docs/en_US/getting-started/getting-started.md]: > Move this block to `Prepare the environment`

Done

**Pull request closed without merging**